prusti-common/src/config.rs

```diff
@@ -380,14 +381,18 @@ pub fn optimizations() -> Optimizations {
     return opt;
 }
 
-/// Enable the purification optimization.
-///
-/// **Note:** this optimization is currently very incomplete and may introduce
-/// unsoudnesses.
+/// Enable purification optimization for impure functions.
 pub fn enable_purification_optimization() -> bool {
     read_setting("enable_purification_optimization")
 }
 
+/// Enable manual axiomatization of pure functions.
+///
+/// **Note:** this is currently very incomplete and may introduce unsoudnesses.
+pub fn enable_manual_axiomatization() -> bool {
+    read_setting("enable_manual_axiomatization")
```

Comment:
I think it currently causes an encoding of the `Nat` domain only.

Reply:
True. I would add `unimplemented!` [here](https://github.com/viperproject/prusti-dev/blob/afa89f858748fced714b72049e79c2f8442fd715/prusti-viper/src/encoder/encoder.rs#L258) because otherwise it can be very confusing why this flag does nothing.